src/intentManager.js

```diff
@@ -35,6 +35,53 @@ define([
         };
     }
 
+    function intentAppIdWidget(mug, options) {
+        var widget = widgets.text(mug, options),
+            input = widget.input,
+            opts = options.vellum.opts().intents,
+            tempSelect = $('<select style="width: 100%" />')
+                .attr("name", widget.id + "-template")
+                .addClass('span3 input-block-level')
+                .append($("<option />").attr("value", "").text("")),
+            templates = {},
+            control = $('<div class="control-row row" />')
+                .append($('<div class="span6" />').append(input))
+                .append($('<div class="span2 control-label">Template</div>'))
+                .append($('<div class="span4" />').append(tempSelect));
```

Comment:
Yeah, I thought of that, but the inner appends (`input` and `tempSelect`) seemed harder to get into a template while retaining a references to the inserted DOM elements. Maybe that's easy and I just don't know it?